src/cmd/singularity/cli/run_help.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package cli
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+
+	"github.com/opencontainers/runtime-tools/generate"
+	"github.com/singularityware/singularity/src/docs"
+	"github.com/singularityware/singularity/src/pkg/buildcfg"
+	"github.com/singularityware/singularity/src/pkg/sylog"
+	"github.com/singularityware/singularity/src/pkg/util/exec"
+	"github.com/singularityware/singularity/src/runtime/engines/config"
+	"github.com/singularityware/singularity/src/runtime/engines/config/oci"
+	"github.com/singularityware/singularity/src/runtime/engines/singularity"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	RunHelpCmd.Flags().SetInterspersed(false)
+
+	SingularityCmd.AddCommand(RunHelpCmd)
+}
+
+// RunHelpCmd singularity run-help <image>
+var RunHelpCmd = &cobra.Command{
+	DisableFlagsInUseLine: true,
+	PreRun:                sylabsToken,
+	Args:                  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+
+		if _, err := os.Stat(args[0]); err == nil {
```

Comment:
What happens when `err != nil`? We need to handle `err` after the conditional here. Some kind of error like `container not found` or something

Reply:
ups, fixing

Reply:
Addressed on f0e95e2